### DISCUSSION TITLE
View organization teams and members on mobile

### DISCUSSION THREAD

**Discussion by jmeridth** [community/community#54256] [Mobile] [answered]
On mobile when I click on an organization I only get the options for repositories and projects.  Would like to be able to click teams and members and then even go to a user's profile.  Currently when I click on a team link in the app it goes to mobile browser view of the org's team.  

Another nit is the mobile web version of teams doesn't show all members, only ones that have a status.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.112.0 (98590521)
Device: iPhone15,2 (iOS 16.4.1)
</details>

**Comment by mattruggio** [comment, marked as answer]
Hey @jmeridth, thank you for your suggestion!  We are always looking to bring parity between our web and mobile experiences so this is good feedback for us.  I can get this filed in our feature request backlog for further review.

**Reply by Nogard-YT** [reply]
This is something that should be available on mobile web version of GitHub as well.
